src/framework/shortcuts/data/shortcuts.xml

```diff
@@ -339,7 +341,7 @@
   <SC>
     <key>pitch-up</key>
     <seq>Up</seq>
-    <ctx>notation-staff-standard</ctx>
+    <ctx>notation-focused</ctx>
```

Comment:
It's better to leave `notation-staff-standard` here, because there are actions for tablature staff on same shortcuts keys

Reply:
I actually was testing. I forgot to change it back. Will do that soon